client/components/common/tce-core/QuestionContainer/index.vue

```diff
@@ -125,7 +126,6 @@ export default {
       }).catch(err => (this.errors = err.inner));
     },
     cancel() {
-      if (!this.editedElement.id) return this.$emit('delete');
```

Comment:
You can't do this due to how question is supposed to behave within standard container